### PR TITLE
Add Commonsearch PageRank Signal

### PR DESCRIPTION
**Pull request by HenriqueLimas**

### CODE REVIEW

File: cosrlib/signals/commonsearch_host_pagerank.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        from the WebDataCommons project.
+    """
+
+    def get_value(self, document, url_metadata):
+
+        rank = url_metadata["domain"].commonsearch_host_pagerank
+        if not rank:
+            return None
+
+        return float(rank)
```
Comment (sylvinus) [inline cosrlib/signals/commonsearch_host_pagerank.py]: The value should be normalized between 0 and 1. (The maximum PageRank value is the one of the first line of the file, and the minimum is 0.15)

**Comment by sylvinus** [comment]
The patch looks great, congrats @HenriqueLimas!

I've added a couple of comments. Also, there are 2 linting errors before we can merge:
https://travis-ci.org/commonsearch/cosr-back/builds/167804916

Thanks!!!

File: cosrlib/signals/commonsearch_host_pagerank.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from . import BaseSignal
+
+
+class Signal(BaseSignal):
+    """ Ranking signal based on the Harmonic Centrality domain ranking
+        from the WebDataCommons project.
+    """
```
Comment (HenriqueLimas) [inline cosrlib/signals/commonsearch_host_pagerank.py]: Yes, I am going to fix that

**Comment by HenriqueLimas** [comment]
@sylvinus, the lint errors is about the Header of the request that I needed to add..Do you think it's correct adding the Header User-Agent?

**Comment by sylvinus** [comment]
Great, thanks a lot @HenriqueLimas !

**Pull request merged**